definitions.d.ts

```diff
@@ -0,0 +1 @@
+declare module "*.svg";
```

Comment:
I'm a bit suspicious of this fix, as its more of a hack than an actual fix, and we lose the type information of the SVG: 
![image](https://user-images.githubusercontent.com/492636/84903488-76abf380-b0ae-11ea-8243-3cbfe6721ed9.png)

Can you try this solution: https://duncanleung.com/import-svg-storybook-webpack-loader/

Reply:
hmm but it's not actually a hack as using typescript with react when importing SVGs require declaring them as a module like this one: https://webpack.js.org/guides/typescript/#importing-other-assets

Reply:
and this one will do the fix for storybook and jsx when we need it, but I will try what you mentioned for sure as I remember that I saw it once before

Reply:
Hmmm weird. Then we can update the definition to specify it as a string at least?

Reply:
it is actually how typescript works with svgs, and other undefined modules, so we declare them them like that but didnt see how to specify them as a string

Reply:
Ok Lets go with this :)